sass-css.md

```diff
@@ -238,52 +231,189 @@ So remember, do not fear multiple classes on elements.
 
+**[[⬆]](#TOC)**
 
-## Do not nest classes in CSS/SASS
+### Adding Third Party CSS
+
+Goes in `vendor` directory, unedited. If there are multiple files, use a directory.
+
+Most included files will be css files so change the extension to `.scss`. If you _do_ have to make changes, keep them small (this is most likely changes to image paths) and do it in a seperate commit.
+
+I prefer to include non-minified source, so that future changes to plugins can be easily seen in a `git diff`.
+
+For example the mixins from [bootstrap-sass](https://github.com/twbs/bootstrap-sass/) go into `vendor/bootstrap/_mixins.scss`.
+
+**[[⬆]](#TOC)**
+
+## <a name="housetrip-style">HouseTrip Style</a>
+
+We use a tweaked bootstrap (v3) grid to layout our pages.
+
+The [bootstrap documentation](http://getbootstrap.com/css/#grid) is the best place to read up on how that works. We can use all of the existing types of bootstrap grid classes: e.g. `col-sm-offset-X`, `.col-sm-push-3`.
+
+It uses (currently) only the `.col-sm-X` classes for the desktop version, we are not ready for full responsivity in our designs, but by following these guidelines _we can get there_.
+
+There are also `.col-xs-X` that apply for all screen widths, including mobile, but use these sparingly.
+
+**[[⬆]](#TOC)**
+
+### Special case div-itis avoidance
+
+When you are simply after the full width of a container, there is no need to do this...
+
+```haml
+.row
+  .col-sm-12
+    %h1
+```
+
+You can simply omit the `row` and `col-sm-12`
+
+```haml
+%h1
+```
+
+Voila.
+
+**[[⬆]](#TOC)**
+
+### Components should flex within the grid element they are inside
+
+A component should be styled independantly of the containing `col-sm-X` `div`. Overall width of a component is provided by the grid, elements within your component can be positioned within _that_ context.
+
+Use floats and absolute positioning where required. You shouldn't need to do anything crazy like using negative margins, if you do find yourself doing it, there's probably another way.
+
+You can also use nested `.row` and `.col-sm-X` classes within your components for additional grid-based layout.
+
+It's also a possible smell if you find yourself setting widths rather than relying on the grid.
+
+**[[⬆]](#TOC)**
+
+### SVG
+
+For all non-photographic images we should try and use SVG. It is XML-based and therefore textand this compresses extremely well.
+
+We can also inline SVG to reduce HTTP requests as we move away from any support other than IE8.
+
+**A note on icon-fonts**
+
+Icon fonts are great. Including *all the icons* is not great. Generating classes for all the icons you need is a recipie for exventual CSS bloat.
+
+We currently have the glyphicons fonts included in v4, but we use only a handful for their stated purpose (tiny icons next to text) and spend a lot of time in CSS tweaking their position.
```

Comment:
Without v4 they have no place. I think only PPP content navigation is still using them. 

I'll do some digging to detail where we are still using them